Source/WebCore/platform/gstreamer/PlatformSpeechSynthesizerGStreamer.cpp

```diff
@@ -95,8 +98,18 @@ GstSpeechSynthesisWrapper::GstSpeechSynthesisWrapper(Ref<PlatformSpeechSynthesiz
         }
     }
 
-    gst_bin_add_many(GST_BIN_CAST(m_pipeline.get()), m_src.get(), audioSink.get(), nullptr);
-    gst_element_link_pads_full(m_src.get(), "src", audioSink.get(), "sink", GST_PAD_LINK_CHECK_NOTHING);
+    m_pitchElement = makeGStreamerElement("pitch", nullptr);
```

Comment:
Here we need to consider the case where `pitch` is not shipped by the runtime. You'd need to emit `WTFLogAlways()` explaining the pitch feature is disabled.

Reply:
Done.